scripts/county_adjacency_msa/index.ts

```diff
@@ -84,6 +108,16 @@ function parseCountyData(data: CountyItem[]): AdjacencyMap {
       );
     }
   }
+
+  const fipsToMsa = parseFipsToMsa();
+  const fipsInMsaList = Object.keys(fipsToMsa);
+
+  fipsInMsaList.forEach(fips => {
+    if (adjacencyMap[fips]) {
+      adjacencyMap[fips].msa_code = fipsToMsa[fips];
+    }
+  });
```

Comment:
Note that if the `msa_code` is `0` it will still show up in the data, maybe you want to remove it? That way you won't need to handle `"0"` later in the front-end code.

```json
{
    "12107": {
      "adjacent_counties": [
        "12001",
        "12007",
        "12019",
        "12035",
        "12083",
        "12109",
        "12127"
      ],
      "msa_code": "0"
    },
}
```